src/scripts/emailBreachAlerts.js

```diff
@@ -0,0 +1,260 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Sentry from "@sentry/nextjs"
+import { acceptedLanguages, negotiateLanguages } from "@fluent/langneg";
+
+import * as pubsub from "@google-cloud/pubsub";
+import * as grpc from "@grpc/grpc-js";
+
+import { getSubscribersByHashes, knexSubscribers } from "../db/tables/subscribers.js";
+import { getEmailAddressesByHashes, knexEmailAddresses } from "../db/tables/emailAddresses.js";
+import { getTemplate } from "../views/emails/email2022.js";
+import { breachAlertEmailPartial } from "../views/emails/emailBreachAlert.js";
+
+import {
+  initEmail,
+  EmailTemplateType,
+  getEmailCtaHref,
+  sendEmail,
+} from "../utils/email.js";
+
+import { initFluentBundles, getMessage } from "../utils/fluent.js";
+import {
+  getAddressesAndLanguageForEmail,
+  getBreachByName,
+  getAllBreachesFromDb,
+  knexHibp
+} from "../utils/hibp.js";
+
+const SENTRY_SLUG = "cron-breach-alerts";
+
+Sentry.init({
+  environment: process.env.APP_ENV,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
```

Comment:
Q: should this be an ENV var so we can tweak it in Heroku/dev envs?

Reply:
I think in this case we want 100%, because it runs hourly and it's under our control (there's only going to be a single script running at a time).

I'd like to eventually move the subscriber out of cron and into e.g. a pool of HTTP workers, we will probably need to reconsider.